blueoil/datasets/camvid.py

```diff
@@ -80,15 +79,13 @@ def files_and_annotations(self):
             text = "val.txt"
 
         filename = os.path.join(self.data_dir, text)
-        df = pd.read_csv(
-            filename,
-            delim_whitespace=True,
-            header=None,
-            names=['image_files', 'label_files'],
-        )
 
-        image_files = df.image_files.tolist()
-        label_files = df.label_files.tolist()
+        image_files, label_files = list(), list()
```

Comment:
Why Don't you use csv module?

Reply:
This is for reading a text file instead of csv file.

Reply:
Python `csv` module can load space separated file.
https://docs.python.org/ja/3/library/csv.html#csv.Dialect.delimiter

It's good to use standard library than implement by ourself.